.travis.yml

```diff
@@ -51,7 +51,7 @@ matrix:
         - os: linux
           env: OPENRCT2_CMAKE_OPTS="-DCMAKE_TOOLCHAIN_FILE=../CMakeLists_mingw.txt -DFORCE32=on" TARGET=windows
         - os: linux
-          env: TARGET=docker64 OPENRCT2_CMAKE_OPTS="-DDISABLE_TTF=ON"
+          env: TARGET=docker64 OPENRCT2_CMAKE_OPTS="-DDISABLE_TTF=ON -DCMAKE_BUILD_TYPE=release"
```

Comment:
Various compiler features and optimisations may change state depending on optimisation level, which can cause errors previously hidden to surface. This will help avoiding such bugs as https://github.com/OpenRCT2/OpenRCT2/pull/4497 which were not caught by Travis.